Dockerfile_cp

```diff
@@ -0,0 +1,21 @@
+# build image
+FROM node:20.7-alpine as builder
+
+WORKDIR /app
+# assuming ./dist is the dist.
+COPY ./dist /app/dist
+
+# production image
+FROM node:20.7-alpine
```

Comment:
Multi-stage build is not required because the only instruction we need to execute in the builder image is `COPY ./dist /app/dist`

A multi-stage build (Dockerfile containing more than one `FROM` statement) has the following advantages:

- parallel build
- smaller image size

These two advantages are not applicable to how `Dockerfile_cp` works. 

Would you remove the multistage build?

Reply:
(done)